canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -58,6 +58,7 @@ class PropertiesMain extends React.Component {
 		this.propertiesController.setLight(props.light);
 		this.propertiesController.setAppData(props.propertiesInfo.appData);
 		this.propertiesController.setExpressionInfo(props.propertiesInfo.expressionInfo);
+		this.propertiesController.setParameterDefId(props.propertiesInfo.parameterDefId);
```

Comment:
Should this just be `id`.  Using parameterDefId makes me think it should go inside the paramDef.